src/api/images.tsx

```diff
@@ -107,3 +109,28 @@ export const createImage = (
       .catch(reject);
   });
 };
+
+export const createImageFromTarball = (
```

Comment:
```suggestion
export const uploadImage = (
```
More concise and just as precise? Also I think we won't always have a tarball here.